less/current/layout/section.less

```diff
@@ -11,4 +11,8 @@ section.page-title-section {
 
 .subsection {
   margin-bottom: 30px;
+}
+.table-scrollbar {
```

Comment:
There is nothing table specific about this selector.
Also we may want to use it on other things rather than just tables,
so it's better rename the selector to just ``.scrollbar``